client/src/components/Verejne/EntitySearch/index.js

```diff
@@ -64,8 +65,8 @@ const EntitySearch = ({
               type="text"
               className="form-control"
               placeholder={FIND_ENTITY_TITLE}
-              value={searchEntityValue}
-              onChange={(e) => setSearchEntityValue(e.target.value)}
+              value={entitySearchValue}
+              onChange={(e) => setEntitySearchValue(e.target.value)}
```

Comment:
Maybe I already wrote it to another PR with this commit, but please, move this to `withHandlers` too.